packages/cli/commands/accounts/clean.js

```diff
@@ -35,7 +35,8 @@ exports.handler = async options => {
 
   trackCommandUsage('accounts-clean', null);
 
-  const filteredTestAccounts = config.portals.filter(p =>
+  const accountsList = config.accounts || config.portals;
```

Comment:
there's also a `getAccounts` util that could help you here

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1177/commits/f9a718f1de95eac66bd6d91f19cf0a3ef74dbce3